embulk-standards/src/main/java/org/embulk/standards/JsonParserPlugin.java

```diff
@@ -207,4 +221,6 @@ private PageBuilder newPageBuilder(Schema schema, PageOutput output) {
     private static final Logger logger = LoggerFactory.getLogger(JsonParserPlugin.class);
 
     private static final Pattern DIGITS_PATTERN = Pattern.compile("\\p{XDigit}+");
+
+    private static final JsonParser JSON_PARSER = new JsonParser();
```

Comment:
It's not an immediate problem because the existing `JsonParser` is immutable, but we may have to change the situation in the future. We can easily forget to update this part in that case.

I'd suggest to declare this as a non-static final member, and initialize it in the constructor. What do you think?

Reply:
From the perspective of current internal implementation of `JsonParser`, creating it multiple times would take a cost because it has `JsonFactory` which is a heavy-weight object.
ref: https://github.com/FasterXML/jackson-docs/wiki/Presentation:-Jackson-Performance#basics-things-you-should-do-anyway
That's why I made it reusable in the class. So creating JsonParser per instance sounds good to me 😃 
(Ideally we should make JsonFactory reusable in JsonParser)